Gemfile

```diff
@@ -22,14 +22,26 @@ gem "haml-rails", '~> 0.9'
 gem 'jquery-rails'
 # Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
 # If we are creating a single page app with vue-router, we should remove this
-gem 'turbolinks', '~> 5'
+gem 'turbolinks', '< 5'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.5'
 # Use Redis adapter to run Action Cable in production
 # gem 'redis', '~> 3.0'
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
 
+# group :assets do
+#   gem 'jquery-datatables-rails', github: 'rweng/jquery-datatables-rails'
+#   gem 'jquery-ui-rails'
+# end
+#
+# gem 'will_paginate'
```

Comment:
You shouldn't add commented-out code and leave it lying around like this. I know there's some stuff commented out in this file already, but that was autogenerated. Plus `jquery-datatables-rails` and `jquery-ui-rails` are repeated below.